src/views/Introduction.js

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+
+import Article from "../components/Article";
+import Graphic from "../components/Graphic";
+import Text from "../components/Text";
+
+const Introduction = ({ view, invert }) => (
+  <section>
+    <Article left view={view} wide={invert}>
+      <Graphic
+        top="20em"
+        left="3em"
+        src="/img/graphics/megan_and_ringo.svg"
+        alt="megan_and_ringo"
```

Comment:
The `alt` attribute specifies alternative text that should be shown if the image is not shown. It should describe the image, not directly state the image filename minus the extension.

For this instance, "Megan and Ringo's Wedding Header."

See this example for more details: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#Alternative_text